### PR TITLE
Copy + adapt travis config from dhtnode to fix upload

### PR DESCRIPTION
**Pull request by mihails-strasuns-sociomantic**

### CODE REVIEW

**Review by nemanja-boric-sociomantic — APPROVED** [review]

**Comment by mathias-lang-sociomantic** [comment]
Looks like you need to fix the pkg file post makd v2

**Comment by mihails-strasuns-sociomantic** [comment]
Yup, just pushed

**Comment by nemanja-boric-sociomantic** [comment]
I've restarted the build, it had the network connectivity issues.

**Comment by nemanja-boric-sociomantic** [comment]
One more try. It can't reach D-APT:

>   Could not connect to master.dl.sourceforge.net:80 (216.105.38.12), connection timed out

**Comment by mihails-strasuns-sociomantic** [comment]
This issue is more interesting but I will have to look into it tomorrow :)

File: .travis.yml [hunk 1]
```diff
@@ -11,24 +11,47 @@ services:
...
+              - beaver dlang make
+              - beaver dlang make pkg
+              # If this is a tag, save the packages in the cache directory so
+              # it's accessible in the Upload Package stage
+              - if test -n "$TRAVIS_TAG"; then
+                    cp -v build/last/pkg/*.deb "$ARTIFACTS_DIR/";
+                fi
+    include:
+        - <<: *test-matrix
+          env: DMD=2.078.3-* F=production
```
Comment (leandro-lucarella-sociomantic) [inline .travis.yml]: Why not `78.*`?

File: pkg/d1to2fix.pkg [hunk 2]
```diff
@@ -1,5 +1,5 @@
-OPTS = dict(
-    name = VAR.name,
+OPTS.update(
```
Comment (leandro-lucarella-sociomantic) [inline pkg/d1to2fix.pkg:1]: Shouldn't this adaptation be done when the MakD submodule is advanced?

Comment (mihails-strasuns-sociomantic) [reply pkg/d1to2fix.pkg:1]: Yes, but that is already merged :shame: Travis script wasn't generating any packages so did not notice makd upgrade has broken it.

File: .travis.yml (hunk 1 above)
Comment (mihails-strasuns-sociomantic) [reply .travis.yml]: Just changed to that in last commit

**Comment by mihails-strasuns-sociomantic** [comment]
Green. Did commit fixup and restarted build again - should be good to go.

**Review by leandro-lucarella-sociomantic — APPROVED** [review]

**Pull request merged**